.pre-commit-config.yaml

```diff
@@ -25,6 +25,11 @@ repos:
   - id: trailing-whitespace
     exclude: *exclude_pre_commit_hooks
 
+- repo: https://github.com/sirosen/check-jsonschema
```

Comment:
Suggesting to add this pre-commit hook, useful to catch silly mistakes sooner when messing with GHA yaml's.

Reply:
Could you do this in a separate PR prlease since it is not related. Should probably also use [`https://github.com/python-checkcheck-jsonschema](https://github.com/python-jsonschema/check-jsonschema) instead since the repo appears to have been moved

Reply:
Yep, done. #6365